USER_GUIDE.md

```diff
@@ -175,9 +175,9 @@ DeleteIndexResponse deleteIndexResponse = client.indices().delete(deleteIndexReq
 
 # Using different transport options
 
-## Amazon Managed OpenSearch
+## Amazon OpenSearch Service
```

Comment:
Update TOC

Should we call these "Managed OpenSearch and OpenSearch Serverless"?

Reply:
The section header could be "Amazon OpenSearch Service and OpenSearch Serverless" though. Would that be better?

Reply:
Yes, so we have 

https://docs.aws.amazon.com/opensearch-service/latest/developerguide/request-signing.html that says "Amazon OpenSearch Service" and https://docs.aws.amazon.com/opensearch-service/latest/developerguide/serverless-clients.html that says "Amazon OpenSearch Serverless" 

So I think that's the correct wording, if you want them combined "Amazon OpenSearch Service" + "Amazon OpenSearch Serverless" = ""Amazon OpenSearch Service and Serverless"?